features/command.feature

```diff
@@ -719,3 +719,114 @@ Feature: WP-CLI Commands
       bar
       """
     And STDERR should be empty
+
+  @bootstrap
```

Comment:
Yes, I agree that is preferable. Now that most of the package have been taken out, it can more easily be contained.